### ISSUE TITLE
isBrowser is false as of webpack@4.16.2

### ISSUE THREAD

**Issue by safareli** [uber/luma.gl#620]
for example on the recent version of webpack we were getting:

error like this:
```
WARNING in ./node_modules/luma.gl/dist/esm/webgl-context/create-headless-context.js

Module not found: Error: Can't resolve 'gl' in '/home/jenkins/workspace/.../node_modules/luma.gl/dist/esm/webgl-context'

 @ ./node_modules/luma.gl/dist/esm/webgl-context/create-headless-context.js

 @ ./node_modules/luma.gl/dist/esm/webgl-context/context.js

 @ ./node_modules/luma.gl/dist/esm/index.js

 @ ./node_modules/@deck.gl/core/dist/esm/lib/layer-manager.js

 @ ./node_modules/@deck.gl/core/dist/esm/index.js

 @ ./node_modules/deck.gl/dist/esm/index.js

```
and it looks like isBrowser from https://github.com/uber/luma.gl/blob/v6.0.0/src/utils/is-browser.js#L12 is false 

related https://github.com/webpack/webpack/issues/7777

**Comment by safareli** [comment]
You are welcome! 
The `isBrowser` being `false` was just an assumption I had but just looking at the error and code, so issue might have been somewhere else.

**Comment by craigbeck** [comment]
still seeing this with `luma.gl@6.0.0`

```
WARNING in ./node_modules/probe.gl/dist/lib/log.js
Module not found: Error: Can't resolve 'asciify-image' in '/Users/craigbeck/dev/premise-frontend-v3/node_modules/probe.gl/dist/lib'
 @ ./node_modules/probe.gl/dist/lib/log.js
 @ ./node_modules/probe.gl/dist/index.js
 @ ./node_modules/@deck.gl/core/dist/esm/utils/log.js
 @ ./node_modules/@deck.gl/core/dist/esm/index.js
 @ ./node_modules/deck.gl/dist/esm/index.js
...

WARNING in ./node_modules/luma.gl/dist/esm/webgl-context/create-headless-context.js
Module not found: Error: Can't resolve 'gl' in '/Users/craigbeck/dev/premise-frontend-v3/node_modules/luma.gl/dist/esm/webgl-context'
 @ ./node_modules/luma.gl/dist/esm/webgl-context/create-headless-context.js
 @ ./node_modules/luma.gl/dist/esm/webgl-context/index.js
 @ ./node_modules/luma.gl/dist/esm/core/animation-loop-proxy.js
 @ ./node_modules/luma.gl/dist/esm/index.js
 @ ./node_modules/@deck.gl/layers/dist/esm/hexagon-cell-layer/hexagon-cell-layer.js
 @ ./node_modules/@deck.gl/layers/dist/esm/index.js
 @ ./node_modules/deck.gl/dist/esm/index.js
...

WARNING in ./node_modules/luma.gl/dist/es5/webgl1/webgl-rendering-context.js
Module not found: Error: Can't resolve 'gl/wrap' in '/Users/craigbeck/dev/premise-frontend-v3/node_modules/luma.gl/dist/es5/webgl1'
 @ ./node_modules/luma.gl/dist/es5/webgl1/webgl-rendering-context.js
 @ ./node_modules/luma.gl/dist/es5/webgl1/polyfill-context.js
 @ ./node_modules/luma.gl/dist/es5/webgl1/index.js
 @ ./node_modules/luma.gl/webgl1.js
 @ ./node_modules/@deck.gl/core/dist/esm/lib/init.js
 @ ./node_modules/@deck.gl/core/dist/esm/index.js
 @ ./node_modules/deck.gl/dist/esm/index.js
...
```

**Comment by ibgreen** [comment]
> still seeing this with luma.gl@6.0.0

@jianhuang01  we need to land the patch and publish 6.0.1

**Comment by ibgreen** [comment]
We may need a patch on probe.gl as well. I have fixed it in the alpha release, but may need a patch on the current release as well.

**Comment by 1chandu** [comment]
@jianhuang01 @ibgreen are there multiple PRs for these? If not all changes ported to 6.0-release, can you do so, moving it to 6.0.2 for now. If all in 6.0.1, please close.

**Comment by ibgreen** [comment]
I think this is fixed in all places and on all branches now.

**Comment by craigbeck** [comment]
👍 confirmed resolved

**Comment by ChunhongZhao** [comment]
Can this issue fix in v5.3? Our project is built on luma.gl^5.3, recently we moved to Webpack 4, the same issue bothers me, thanks in advance.

**Comment by 1chandu** [comment]
`5.3.1` published with #760  @ChunhongZhao
